### ISSUE TITLE
Is it still working?

### ISSUE THREAD

**Issue by bblanchon** [ZitRos/save-analytics-from-content-blockers#1]
Hi @ZitRos,

Thanks for this excellent and unique tutorial.
I tried to implement this solution with the only difference that I used NGINX for the proxy.
Everything looks fine excepts that I don't see any visits in the Google Analytics console.

I can see that the "collect" request and the 200 response in the Developer Tools, but the [Tag Assistant](https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk?hl=fr) says "No HTTP response detected."

I'm pretty sure I did everything as told, and I don't believe the proxy would make any difference.
So, I'm wondering if this technique still works according to your own experience?

Regards,
Benoit

**Comment by ZitRos** [comment]
Hello!

Just checked my website; the extension you've mentioned says `No HTTP response detected.`, while in Google Analytics I see that the page event is recorded indeed.

I noticed this behavior earlier, but didn't find any consequences of this error.

Anyway, are you using the `analytics.js` script included in this repo? I think Google could update this script at some point.

If you find a solution to `No HTTP response detected.` at least, please let me know!

**Comment by bblanchon** [comment]
Hi @ZitRos,

Thank you very much for your answer.

Since it works on your side, it could only mean one thing: there is something wrong on mine.
So I repeated all the tests I performed yesterday but with another account and it worked!

It turns out that during my experimented I broke a filter in the Google Analytics view.
Removing the filter solved my problem.

Sorry to have bothered you, and thanks again.

Regards,
Benoit
